### PR TITLE
Add the information of Godwoken testnet_v1

### PR DESCRIPTION
**Pull request by Flouse**
## Godwoken testnet_v1 Web3 RPC
https://godwoken-testnet-web3-v1-rpc.ckbapp.dev

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,24 @@
...
 
+* [How to test your Solidity contracts on Godwoken v1 using Hardhat](https://github.com/nervosnetwork/godwoken-tests)
 * [Godwoken Document](https://docs.godwoken.io)
-* [Migrate EVM contract to polyjuice](https://github.com/nervosnetwork/godwoken/blob/master/docs/migrate_evm_contract_to_polyjuice.md)
+* [Migrate EVM contract to Polyjuice (v0)](https://github.com/nervosnetwork/godwoken/blob/master/docs/migrate_evm_contract_to_polyjuice.md)
+
+## Web3 provider (v0)
+
+> Notice: 
+This provider was deprcated in Godwoken v1.
```
Comment (e00dan) [inline README.md]: deprecated

**Review by e00dan — APPROVED** [review]

**Comment by zhangyouxin** [comment]
Maybe contain eth_account_lock info in this pr too？

File: README.md (hunk 1 above)
Comment (Flouse) [inline README.md]:
```suggestion
This provider was deprecated in Godwoken v1.
```

File: testnet_v1/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+## Godwoken testnet_v1
+
+* [Readonly node config](./gw-testnet_v1-config-readonly.toml)
+* Explorer: TBD
```
Comment (Flouse) [inline testnet_v1/README.md]:
```suggestion
* Explorer: https://v1.aggron.gwscan.com
```

**Review by jjyr — APPROVED** [review]

**Pull request merged**